services/web/pkg/config/defaults/defaultconfig.go

```diff
@@ -47,9 +47,34 @@ func DefaultConfig() *config.Config {
 					ResponseType: "code",
 					Scope:        "openid profile email",
 				},
-				Apps: []string{"files", "search", "preview", "text-editor", "pdf-viewer", "external", "user-management"},
+				Apps: []string{"files", "search", "text-editor", "pdf-viewer", "external", "user-management"},
+				ExternalApps: []config.ExternalApp{
+					{
+						ID:   "preview",
+						Path: "web-app-preview",
+						Config: map[string]interface{}{
+							"mimeTypes": []string{
```

Comment:
We only need the mime types which are missing in https://github.com/owncloud/web/blob/904be00790e57a66bab95bd488bfe7509a10dc28/packages/web-app-preview/src/index.js#L26
So this list here only needs your new mime types:
```
"image/tiff",
"image/bmp",
"image/x-ms-bmp",
```

Reply:
Ah good to know! :+1: